proto/ibc/applications/interchain_accounts/v1/genesis.proto

```diff
@@ -1,11 +1,26 @@
 syntax = "proto3";
+
 package ibc.applications.interchain_accounts.v1;
 
 option go_package = "github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/types";
 
 import "gogoproto/gogo.proto";
 
-// GenesisState defines the interchain_account genesis state
+// GenesisState defines the interchain accounts genesis state
 message GenesisState {
-  string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
+  repeated ActiveChannel               active_channels     = 1 [(gogoproto.moretags) = "yaml:\"active_channels\""];
+  repeated RegisteredInterchainAccount interchain_accounts = 2 [(gogoproto.moretags) = "yaml:\"interchain_accounts\""];
```

Comment:
Was thinking about this too actually! I think it would easier to do after the sub package split. I guess the only difference would be `repeated string port_ids` for the controller and a single port ID for the host.